assets/js/signup.js

```diff
@@ -0,0 +1,176 @@
+$(function(){
+
+	var stepStack = [];
```

Comment:
The back button is currently broken for the "js-enabled" version. This could be resolved (at least for modern browsers) by using the browser History API

Reply:
wontfix yet